TumbleBitSetup.Tests/ProtocolTests.cs

```diff
@@ -399,12 +395,9 @@ public bool _ShortN(BigInteger Exp, int shortKeySize, int longKeySize)
             // A case where "shortKeySize"-bits N is used for proving and "longKeySize"-bits is needed for verifying.
             var keyPair = new RsaKey(Exp, shortKeySize);
 
-            var privKey = keyPair._privKey;
-            var pubKey = new RsaPubKey(keyPair);
-
-            byte[][] signature = PermutationTest.Proving(privKey.P, privKey.Q, privKey.PublicExponent, alpha, ps);
+            var signature = keyPair.CreatePermutationTest(setup);
             // passing "longKeySize" as the keySize.
-            return PermutationTest.Verifying(pubKey, signature, alpha, longKeySize, ps);
+            return keyPair.PublicKey.VerifyPermutationTestProof(signature, setup);
```

Comment:
This is not quite the same as what this test indicated, because now with this new way of passing the arguments to the Verification function, we assume the the "keyLength" system parameter is the actual length of the publicKey that we pass. So an attacker will not be able to pass a "keyLength" that is different than the actual length of the publicKey used for verification ( you can see [here](https://github.com/osagga/TumbleBitSetup/pull/23/files#diff-02435bfa7e267216b41d7e93aaf9ffd2R42) that the keyLength that is passed is the actual keyLength.) So @goldbe, I think we should remove this unit test and maybe modify the "System parameters"
for the verification algorithm (in setup.pdf) in PermutationTest to not take the keyLength as an independent parameter, given how in this new setup, the keyLength will always be dependent on the PublicKey (N, e).